### PR TITLE
[lockscreen] make lockscreen image actually visible

### PR DESCRIPTION
**Pull request by Morpog**

### CODE REVIEW

**Comment by rburchell** [comment]
nicer alternative: try invert the parent child relationship, so the Rectangle is the parent, and the lockscreen Image is on top of it (no z setting necessary, then)

**Comment by Morpog** [comment]
Hope thats better and I didn't messed with history. Still new to git handling.

File: src/qml/Lockscreen.qml [hunk 1]
```diff
@@ -1,12 +1,14 @@
 import QtQuick 2.0
 
-Image {
+Rectangle {
     id: lockScreen
-    source: "file://" + wallpaperSource.value
+    color: "black"
+    anchors.fill: parent
```
Comment (rburchell) [inline src/qml/Lockscreen.qml]: this wasn't there previously. is Lockscreen sized by its creator? (thus making this unneeded)

**Comment by Morpog** [comment]
I give up for now :)

**Pull request closed without merging**